### DISCUSSION TITLE
Full Text Search reserved characters

### DISCUSSION THREAD

**Discussion by mcrio** [ravendb/ravendb#14216] [Q&A] [answered]
Hi, when using `search` are there any reserved characters other than `*`?

If yes, is there a method to escape special characters, or are they ignored by the analyzers?

**Comment by garayx** [comment, marked as answer]
hi

On `Search()` method only the `*` is reserved. We are not using analyzer on wildcard queries, as far as I know. 

You can see more info on this case here:
https://ayende.com/blog/191841-B/understanding-query-processing-and-wildcards-in-ravendb
